src/pages/patterns/gathering-information.js

```diff
@@ -95,11 +122,74 @@ export default () => (
           <Cite>
             View{" "}
             <Link href="https://github.com/barnardos/design-system/issues/404">
-              asking for names research
+              names research
             </Link>{" "}
             on GitHub.
           </Cite>
-
+          <Heading>Addresses</Heading>
+          <Subheading>Address lookup</Subheading>
+          <Paragraph>Where possible use an address lookup.</Paragraph>
+          <Figures>
+            <Figure>
+              <TextInput
+                id="text-input-post-code-example"
+                label="Postcode (optional)"
+                size={16}
+              />
+              <Submit>Find address</Submit>
+            </Figure>
+          </Figures>
+          <Cite>
+            View{" "}
+            <Link href="https://github.com/barnardos/design-system/issues/341">
+              address lookup research
+            </Link>{" "}
+            on GitHub.
+          </Cite>
+          <Subheading>Single input</Subheading>
+          <Paragraph>
+            Where possible use a single Text Area to accommodate the broadest
+            range of address formats.
+          </Paragraph>
+          <Figures>
+            <Figure>
+              <TextArea
+                id="text-input-first-name-example"
+                label="Full address"
+                size={48}
+                verticalSize={4}
+              />
+            </Figure>
+          </Figures>
+          <Subheading>Multiple inputs</Subheading>
+          <Paragraph>
+            If your product needs to reliably extract parts of the address, use
+            multiple Text Inputs.
```

Comment:
We should link off to research for collecting address with multiple address inputs, such as [gov.uk's address pattern](https://design-system.service.gov.uk/patterns/addresses/#when-to-use-multiple-text-inputs).

Reply:
ids are updated, research links are added and branch has been rebased.